spec.bs

```diff
@@ -3204,6 +3190,20 @@ dictionary ReportWinBrowserSignals : ReportingBrowserSignals {
     bidding signals server
 </dl>
 
+<xmp class="idl">
+dictionary DirectFromSellerSignalsForBuyer {
+  DOMString? auctionSignals = null;
+  DOMString? perBuyerSignals = null;
+};
+</xmp>
+
+<xmp class="idl">
+dictionary DirectFromSellerSignalsForSeller {
+  DOMString? auctionSignals = null;
+  DOMString? sellerSignals = null;

```

Comment:
The worklet functions are expecting that perBuyerSignals / auctionSignals / sellerSignals are the result of turning the JSON strings into values (objects, arrays, numbers, strings, true, false, null). But, IIUC, we're only passing the JSON strings to worklets?

Reply:
forget that again :(

Reply:
I know for classic auctionSignals / sellerSignals / perBuyerSignals, we invoke `[=parsing a JSON string to a JavaScript value=]` [0], but in that case, we're just using the result as a function argument -- we're not putting into a dict.

[0] https://github.com/WICG/turtledove/blob/ec23844e4211746ca009a75185d826c2dc1f6a67/spec.bs#L2275

Reply:
This question is about https://github.com/WICG/turtledove/pull/774/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR1917-R1922, right?

I think what that is doing is OK, as long as the parsed value isn't a platform object (something inheriting from an interface in Web IDL), since everything else like, like dictionaries/maps are interchangeable from a Web IDL/infra perspective.